diffpy/pdfmorph/morphs/morph.py

```diff
@@ -24,11 +24,11 @@
 
 
 class Morph(object):
-    '''Base class for implementing a morph on an objective given a reference.
+    '''Base class for implementing a morph given a target.
 
     Adapted from diffpy.pdfgetx to include two sets of arrays that get passed
-    through. In most cases, the objective is modified by a morph, but it is
-    acceptable for morph the reference as well, such as to change the range of
+    through. In most cases, only the morph is modified, but it is
```

Comment:
Let's remove this sentence.  Changing the range of an array is not a morphing operation.

Reply:
okie